testing/rtrlib/APKBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: Christian Franke <nobody@nowhere.ws>
+pkgname=rtrlib
+pkgver=0.6.3
+pkgrel=0
+pkgdesc="An open-source C implementation of the RPKI/Router Protocol client"
+url="https://github.com/rtrlib/rtrlib"
+arch="all"
+license="MIT"
+depends=""
+makedepends="cmake cmocka-dev doxygen graphviz libssh-dev"
+install=""
+subpackages="$pkgname-dev $pkgname-doc"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/rtrlib/$pkgname/archive/v$pkgver.tar.gz"
```

Comment:
better to remove `$pkgname` from URL to easy navigate to source

Reply:
Done.